test/identify-test.es6

```diff
@@ -34,6 +34,21 @@ FORMAT: 1A
     });
   });
 
+  describe('with Windows-style line breaks and arbitrary valid content', () => {
+    const source = `\r
```

Comment:
I think windows line break is \n\r and in this case it would be \r\n, wouldn't it?